nupic/research/frameworks/dendrites/modules/dendritic_layers.py

```diff
@@ -173,43 +194,24 @@ class GatingDendriticLayer2d(DendriticLayer2dBase):
     multiplied by a single value computed via dendrites.
     """
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.dendritic_gate_2d = DendriticGate2d()
+
     def apply_dendrites(self, y, dendrite_activations):
-        """
-        Returns the output of the gating convolutional dendritic layer by multiplying
-        all values in each output channel by the selected dendrite activations.
-        Dendrite activations are selected based on the maximum activations across all
```

Comment:
Why remove all the documentation here?

Reply:
The documentation is with the function `dendritic_gate_2d`. I figured it'd be best to keep it in one place for maintainability.

Reply:
The documentation is with the function `dendritic_gate_2d`. I figured it'd be best to keep it in one place for maintainability.